tools/kubectl/plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: 0.1
+downloads:
+  - name: kubectl
+    downloads:
+      - os:
+          linux: linux
+          macos: darwin
+        cpu:
+          x86_64: amd64
+          arm_64: arm64
+        url: https://dl.k8s.io/release/v${version}/bin/${os}/${cpu}/kubectl
```

Comment:
Can you also add the Windows release https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/